### DISCUSSION TITLE
How to add AnonAddy as Yahoo Mailbox?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [anonaddy/anonaddy#513] [Q&A] [answered]
https://uk.mail.yahoo.com/d/settings/1 or https://play.google.com/store/apps/details?id=com.yahoo.mobile.client.android.mail&pcampaignid=web_share

![image](https://github.com/anonaddy/anonaddy/assets/42837531/5df792b3-9f7a-44cf-9e95-282dcb50a053)

**Comment by willbrowningme** [comment, marked as answer]
You cannot add your aliases to your email client etc.

addy.io does not provide SMTP credentials or allow access via IMAP / POP3.

**Reply by roke-julian-lockhart** [reply]
However, I can't add it as a reply-to address without doing so.

![image](https://github.com/anonaddy/anonaddy/assets/42837531/26daedea-101b-483a-a211-86d03f1a475c)

**Comment by willbrowningme** [comment]
https://addy.io/help/replying-to-email-using-an-alias/
